### ISSUE TITLE
Extension stopped working after last update

### ISSUE THREAD

**Issue by environmentalcode0** [micheleg/dash-to-dock#2077]
I'm on ArchLinux with GNOME 44.3 and after I updated dash-to-dock to v87 the extension doesn't work anymore. If I click on dash-to-dock settings from the Extensions app I get the following:

```The settings of extension dash-to-dock@micxgx.gmail.com had an error:
SyntaxError: import declarations may only appear at top level of a module
Stack trace:
_init@resource:///org/gnome/Shell/Extensions/js/extensionPrefsDialog.js:23:33
ExtensionPrefsDialog@resource:///org/gnome/Shell/Extensions/js/extensionPrefsDialog.js:10:4
OpenExtensionPrefsAsync@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:124:33
async*_handleMethodCall@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:373:35
_wrapJSObject/<@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:408:34
run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:186:20
main@resource:///org/gnome/Shell/Extensions/js/main.js:22:13
run@resource:///org/gnome/gjs/modules/script/package.js:206:19
start@resource:///org/gnome/gjs/modules/script/package.js:190:8
@/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17
```

**Comment by D3vil0p3r** [comment]
Same issue here. On Extensions is reported that Ubuntu Dock is not compatible with my current GNOME version (44.3).

**Comment by vanvugt** [comment]
Version 85 and later has dropped support for GNOME versions older than 45. This is sad, but we have been forced into it by an architectural change in GNOME 45.

```
commit 2ec2e0e284f6898692f7733997959b16244c16ef
Author: Marco Trevisan (Treviño) <mail@3v1n0.net>
Date:   Mon Aug 14 15:50:40 2023 +0200

    extension: Port to ESM
    
    Use ESM imports as gnome-45 requires, so drop compatibility with all the
    old versions so that we could modernize things a bit in future.
    
    Closes: #2066
```

Per [the download site](https://extensions.gnome.org/extension/307/dash-to-dock/), GNOME 44 users are limited to extension version 84.
